src/plugins/media-analysis/media-analysis-utils.js

```diff
@@ -54,6 +65,65 @@ export function createDefaultContent(t) {
   };
 }
 
+export function exportChaptersToCsv(chapters) {
+  return [
+    'startPosition,title,color,text',
+    ...chapters.map(chapter => [
+      chapter.startPosition.toString(),
+      `"${chapter.title.replace(/"/g, '""')}"`,
+      `"${chapter.color.replace(/"/g, '""')}"`,
+      `"${chapter.text.replace(/"/g, '""')}"`
+    ].join(',')),
+    ''
+  ].join('\n');
+}
+
+export function importChaptersFromCsv(csvStringOrFile) {
+  return new Promise((resolve, reject) => {
+    parseCsv(csvStringOrFile, {
+      header: true,
+      skipEmptyLines: true,
+      error: error => reject(Object.assign(new Error(), error)),
+      complete: ({ data, errors }) => {
+        if (errors.length) {
+          return reject(Object.assign(new Error(), errors[0]));
+        }
+
+        const chapters = [];
+        try {
+          let lastStartPosition = 0 - Number.EPSILON;
+          for (let index = 0; index < data.length; index += 1) {
+            const row = data[index];
+            const chapter = {
+              key: uniqueId.create(),
+              startPosition: Number(row.startPosition),
+              color: row.color,
+              title: row.title,
+              text: row.text
+            };
+            joi.attempt(chapter, chapterSchema, { abortEarly: false, convert: false, noDefaults: true });
+            if (index === 0 && chapter.startPosition !== 0) {
+              throw new Error('First chapter has to start at position 0');
```

Comment:
Should we maybe pass `t` in here to throw translated error messages?

Reply:
Do you think it's worth it? These are just three out of many things that can go wrong (and have only English error messages coming from the libraries). Also they can never happen as long as users use Anavis to create the CSV. They could only occurr when the file has been meddled with manually in which case I assume a "power user". I didn't want to bloat our translation file just because of this super edge case (probably not used by more than 2-3 people) feature. What do you think?

Reply:
Ok then, you're right, let's not invest so much in this relatively hidden feature.